lms/static/js/dashboard/views/course_list_view.js

```diff
@@ -0,0 +1,40 @@
+;(function (define) {
+    'use strict';
+
+    define(['jquery', 'common/js/components/views/list'], function ($, ListView) {
+
+        var CourseListView = ListView.extend({
+
+            initialize: function (options) {
+                this.itemViewClass = options.itemViewClass || this.itemViewClass;
+                this.template = options.template;
+                this.parent = options.parent;
+
+                this.itemViews = [];
+            },
+
+            render: function () {
+                var self = this;
+
+                this.$el.html($(this.template).html());
+                this.$('.course').each(function () {
+                    self.createItemView(this);
```

Comment:
Using ```this``` here is confusing, please update to use the value passed into the function ( see https://api.jquery.com/each/)